lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -982,11 +1071,25 @@ def add(
     if isinstance(state, str):
         state = SnapState(state)
 
-    return _wrap_snap_operations(snap_names, state, channel, classic, devmode, cohort, revision)
+    return _wrap_snap_operations(
+        snap_names=snap_names,
+        state=state,
+        channel=channel if channel is not None else "",
```

Comment:
```suggestion
        channel=channel or "",
```

seems to be equivalent, wdyt?